charts/super-agent/charts/super-agent-deployment/templates/_helpers.tpl

```diff
@@ -283,3 +283,116 @@ value is provided, it defaults to `""` (empty string) so this helper can be used
   {{- /* Empty string */ -}}
 {{- end -}}
 {{- end -}}
+
+{{/* check if both L1 ClientID and ClientSecret are provided */}}
+{{- define "newrelic-super-agent.auth.l1Identity" -}}
+{{- if and (include "newrelic-super-agent.auth.l1IdentityClientId" .) (include "newrelic-super-agent.auth.l1IdentityClientSecret" .) -}}
+    true
```

Comment:
you can decide if you want to fail or show any message if just one is provided.
Or write a mesasge to the cusotmer depending on which auth is enabled

Reply:
we can also decide to fail if old and new auth is configured at the same time

Reply:
I added the check in the template, not in the helper. So basically is:
* at least one method should be provided: userKey or clientId/clientSecrert, it will fail if none
* if both are provided env variables for both are created and only clientId/clientSecret will be used in the shell script

sdyt?